softlayer/stemcell/softlayer_stemcell_test.go

```diff
@@ -26,14 +27,18 @@ var _ = Describe("SoftLayerStemcell", func() {
 		stemcell = NewSoftLayerStemcell(1234, "fake-stemcell-uuid", DefaultKind, softLayerClient, logger)
 	})
 
-	Describe("Delete", func() {
+	Describe("delete after active transaction finishes", func() {
```

Comment:
By convention we always have either a `Context` or `Describe` with the name of the method under test prefixed with `#` so for this case, I'd suggest something like:
```go
Context("SoftlayerStemcell", func() {
  Describe("#Delete", func() {
  })
...
})
```